### PR TITLE
Remove something

### PR DESCRIPTION
**Pull request by alexey-milovidov**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

File: src/IO/WriteBufferFromS3TaskTracker.cpp [hunk 1]
```diff
@@ -121,17 +121,17 @@ void WriteBufferFromS3::TaskTracker::add(Callback && func)
     /// preallocation for the second issue
     FinishedList pre_allocated_finished {future_placeholder};
 
-    Callback func_with_notification = [&, func=std::move(func), pre_allocated_finished=std::move(pre_allocated_finished)] () mutable
+    Callback func_with_notification = [&, my_func = std::move(func), my_pre_allocated_finished = std::move(pre_allocated_finished)]() mutable
```
Comment (alexey-milovidov) [inline src/IO/WriteBufferFromS3TaskTracker.cpp:124]: I've installed the trunk clang on my machine.

**Comment by robot-clickhouse** [comment]
This is an automated comment for commit 956c399b2af27e6d70c2e4894657f010efb2594d with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/50411/956c399b2af27e6d70c2e4894657f010efb2594d/ci_running.html)
The overall status of the commit is 🟡 pending

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td>🟡 pending</td></tr>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td>🟢 success</td></tr>
</table>

File: src/Storages/StorageDistributed.cpp [hunk 2]
```diff
@@ -1137,13 +1137,6 @@ SinkToStoragePtr StorageDistributed::write(const ASTPtr &, const StorageMetadata
     auto cluster = getCluster();
     const auto & settings = local_context->getSettingsRef();
 
-    /// Ban an attempt to make async insert into the table belonging to DatabaseMemory
```
Comment (alexey-milovidov) [inline src/Storages/StorageDistributed.cpp:1140]: Memory database have supported storing data for tables for a long time already.

File: src/Storages/StorageS3.h [hunk 3]
```diff
@@ -31,7 +31,6 @@ namespace DB
 {
 
 class PullingPipelineExecutor;
-class StorageS3SequentialSource;
```
Comment (alexey-milovidov) [inline src/Storages/StorageS3.h:34]: No such class.

**Pull request merged**